.swift-version

```diff
@@ -0,0 +1 @@
+3.0
```

Comment:
This file is used in the upcoming Cocoapods version 1.1.0 to identify the swift version